H_ecto-models.md

```diff
@@ -517,6 +517,77 @@ end
 
 That's the end of our walk-through of Ecto usage in our controller actions. There is quite a bit more that Ecto models can do. Please take a look at the [Ecto documentation](http://hexdocs.pm/ecto/) for the rest of the story.
 
+### Data Relationship and Dependencies
+
+Handling individual tables is great, but if we want to build a modern web application, we will need a way to relate our data to each other. If we were Ruby on Rails refugees, we'll be glad to know Ecto provides a very familiar API for building relationships between models:
+
+`Schema.has_many/3` declares one to many relationships, for example, if we were building Youtube, our user model might have many uploaded video models.
+`Schema.belongs_to/3` declares a one to one relationship between parent and children models. In the Youtube example, an uploaded video belongs to its user.
+`Schema.has_one/3` declares a one to one relationship. Note that has_one is just like has_many except instead of returning a collection of model structs, we get just one model struct. Continuing with the Youtube example, while an user might have many uploaded videos, the user might only have one featured video.
+
+```elixir
+# models/user.ex
+defmodule HelloPhoenix.User do
+. . .
+  schema "users" do
+    field :name, :string
+    field :email, :string
+    field :bio, :string
+    field :number_of_pets, :integer
+
+    has_many :videos, HelloPhoenix.Video
+    timestamps
+  end
+. . .
+end
+
+# models/video.ex
+defmodule HelloPhoenix.Video do
+. . .
+  schema "videos" do
+    field :name, :string
+    field :approved_at, Ecto.DateTime
+    field :description, :text
+    field :likes, :integer
+    field :views, :integer
+
+    # If we don't provide a foreign_key, Ecto
+    # guesses it as the atom plus _id
+    belongs_to :user, HelloPhoenix.User, foreign_key: :user_id 
+
+    # If our table's column for indictating when 
+    # a row was first inserted is named "created_at", 
+    # we'll need to let Ecto know. ActiveRecord 
+    # immigrants in particular will need to do this as a 
+    # signal of allegience to Ecto
+    timestamps inserted_at: :created_at
+  end
+  @required_fields ~w(name approved_at description user_id)
+. . .
+end
+```
+Note that we don't declare the field `user_id` in the video model, but, as a required (or optional) field, we do declare the `user_id` field.
+
+We can use our newly declared relationships in our controllers as thus:
```

Comment:
What would you think about changing the "as thus" to "like this"? To my ear, this seems more natural.